src/studio/src/designer/frontend/ux-editor/features/serviceConfigurations/manageServiceConfigurations/manageServiceConfigurationSagas.ts

```diff
@@ -82,7 +71,7 @@ export function* saveServiceConfigurationSaga(): SagaIterator {
         return acc;
       }, {});
 
-    yield call(post, getSaveServiceConfigurationUrl(), {
+    yield call(post, getSveSerConfUrl(), {
```

Comment:
why the rename?

Reply:
File length was breaking the line in the parent file... so a rename made the code look more readable.

Reply:
Does it though? Maybe renaming the function to something that we can understand?